Cabal/Distribution/PackageDescription/PrettyPrint.hs

```diff
@@ -285,11 +285,12 @@ showPackageDescription pkg = render $
   $$ (case library pkg of
         Nothing -> mempty
         Just lib -> ppLibrary' lib)
-  $$ vcat [ space $$ ppLibrary' lib | lib <- subLibraries pkg ]
+  $$ vcat [ space $$ ppSubLibrary lib | lib <- subLibraries pkg ]
```

Comment:
As above. Is this actually just an independent fix for the pretty printer or is there some connection to this patch that I'm missing?